bus-tools/recv-bus-event.py

```diff
@@ -22,21 +22,36 @@
 
 from kombu.mixins import ConsumerMixin
 from pprint import pformat
+from kombu import binding as Binding
 
-EXCHANGE = kombu.Exchange('xivo', type='topic')
 
 logging.basicConfig(level=logging.INFO, format='%(asctime)s [%(process)d] (%(levelname)s) (%(name)s): %(message)s')
 logger = logging.getLogger(__name__)
 
 
 class C(ConsumerMixin):
 
-    def __init__(self, connection, routing_key):
+    def __init__(self, connection, routing_key, bindings, tenant):
         self.connection = connection
         self.routing_key = routing_key
+        self.bindings = bindings
+        self.tenant = tenant
 
     def get_consumers(self, Consumer, channel):
-        return [Consumer(kombu.Queue(exchange=EXCHANGE, routing_key=self.routing_key, exclusive=True),
+        bindings = []
+        if self.routing_key:
+            print('Use routing key binding')
+            exchange = kombu.Exchange('xivo', type='topic')
+        if self.bindings:
+            print('Use headers binding')
+            exchange = kombu.Exchange('wazo-headers', type='headers')
+            for event in self.bindings.split(','):
+                bindings.append(kombu.binding(
```

Comment:
If self.tenant is None (omitted --tenant on the CLI), the consumer will only listen for events that have the key `tenant_uuid` = None and any other value or absence of the key will be ignored

```suggestion
                arguments = {'name': event}
                if self.tenant is not None:
                    arguments.update(tenant_uuid=self.tenant)
                bindings.append(kombu.binding(
```

Unless that is the intended behavior